osi_common.proto

```diff
@@ -215,6 +215,39 @@ message Identifier
     optional uint64 value = 1;
 }
 
+// \brief References to external objects
+//
+// The external reference is used to identify objects defined in non-OSI
+// descriptions. This could be other OpenX-Standards as well as user defined
+// simulation environments.
+//
+message ExternalReference
+{
+    // The source of the external references
+    //
+    // Defines the original source of an object as an URI.
+    // The URI should be follow the syntax according to 
+    // \link https://tools.ietf.org/html/rfc3986 RFC 3986\endlink.
+    //
+    optional string reference = 1;
```

Comment:
Include URI as a suggestion instead of a must, considering that it can also be a model-reference.
Take a look at the map, and model references. 
Possibly to adapt this definition also to model and map references.